x-pack/plugins/maps/public/components/map/mb/view.js

```diff
@@ -9,20 +9,29 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { ResizeChecker } from 'ui/resize_checker';
 import { syncLayerOrder, removeOrphanedSourcesAndLayers, createMbMapInstance } from './utils';
-import { DECIMAL_DEGREES_PRECISION, ZOOM_PRECISION } from '../../../../common/constants';
+import { DECIMAL_DEGREES_PRECISION, FEATURE_ID_PROPERTY_NAME, ZOOM_PRECISION } from '../../../../common/constants';
 import mapboxgl from 'mapbox-gl';
 import { FeatureTooltip } from '../feature_tooltip';
 
+
+const TOOLTIP_TYPE = {
+  HOVER: 'HOVER',
+  LOCKED: 'LOCKED'
+};
+
 export class MBMapContainer extends React.Component {
 
   constructor() {
     super();
     this._mbMap = null;
     this._tooltipContainer = document.createElement('div');
     this._mbPopup = new mapboxgl.Popup({
-      closeButton: false,
+      closeButton: true,
```

Comment:
Instead of using mapbox's close button, which is really small and does not look like the EUI close button, how about updating FeatureTooltip to render the close button?

Reply:
yes, makes sense. will do.